ddtrace/llmobs/_experiment.py

```diff
@@ -128,8 +137,71 @@ def run(self, jobs: int = 1, raise_errors: bool = False, sample_size: Optional[i
         self._run_evaluators(task_results, raise_errors=raise_errors)
         return
 
-    def _run_task(self, jobs: int, raise_errors: bool = False, sample_size: Optional[int] = None) -> List[Any]:
-        return []
+    def _process_record(self, idx_record: Tuple[int, DatasetRecord]) -> Dict[str, JSONType]:
+        if not self._llmobs_instance or not self._llmobs_instance.enabled:
```

Comment:
i think we should info log on the enabled condition and `raise` on the instance condition

Reply:
Will do. Note that for `_process_record/_run_task()`, I'm not actually raising/logging because these should never be called by users directly and is already handled in `run()`.